test/src/e2e_vm_tests/test_programs/should_pass/stdlib/contract_id_type/src/main.sw

```diff
@@ -9,8 +9,8 @@ fn main() -> bool {
     let id_2 = ~ContractId::from(0x0000000000000000000000000000000000000000000000000000000000000000);
 
     assert(id_1 == id_2);
-    assert(id_1 == ~ContractId::from(BASE_ASSET_ID));
-    assert(id_1.into() == BASE_ASSET_ID);
+    assert(id_1 == BASE_ASSET_ID);
+    assert(id_1.into() == BASE_ASSET_ID.into());
```

Comment:
I think you can remove this line. Previously the `BASE_ASSET_ID` was a constant instead of the `ContractId` type so `.into()` was needed. This is no longer the case as the test above covers it.

Reply:
Yeah, you are right. What do you think about changing this line to below line to make testing into()?
assert(0x0000000000000000000000000000000000000000000000000000000000000000 == BASE_ASSET_ID.into());

Reply:
Done